burn-book/src/getting-started.md

```diff
@@ -32,7 +32,7 @@ then add the dependency with
 cargo add burn --features wgpu
 ```
 
-and compile it by executing
+add `burn-wgpu = "0.10.0"` in Cargo.toml, and compile it by executing
```

Comment:
If we change line 32 for 
`cargo add burn burn-wgpu` it will add both dependancies at the same time and ensure the latest version is fetched